### ISSUE TITLE
inconsistent roles in authorizationChecker depending on where @Authorized decorator is placed

### ISSUE THREAD

**Issue by mogusbi** [pleerock/routing-controllers#308]
Depending on where you place the `@Authorized` decorator on your controller, the value of the role passed in is returned as either a string or an array which is a bit inconsistent

Take this as an example

```typescript
@JsonController('/string')
@Authorized('Role')
export class StringController {
  @Get()
  public httpGet () {
    return 'Hello world';
  }
}

@JsonController('/array')
export class ArrayController {
  @Get()
  @Authorized('Role')
  public httpGet () {{
    return 'Hello world';
  }
}

function authorizationChecker (action, role) {
  console.log(role);
}
```

The `StringController` will output `Role` inside the `authorizationChecker` whilst the `ArrayController` will return `['Role']`. Shouldn't the value of the role remain the same no matter where the decorator is placed?

This is using `routing-controllers@0.7.5`, `express@4.16.2` and `node@8.2.1`

**Comment by NoNameProvided** [comment]
You are right it should be an array all the time.

**Comment by 19majkel94** [comment]
Should be fixed in 0.7.7 (to be released) by 8cc84b0 😉
